Mathlib/RingTheory/TensorProduct.lean

```diff
@@ -1017,11 +1017,11 @@ end
 
 section Basis
 
-variable {k : Type _} [CommRing k] (R : Type _) [Ring R] [Algebra k R] {M : Type _}
-  [AddCommMonoid M] [Module k M] {ι : Type _} (b : Basis ι k M)
-
 -- porting note: need to make a universe explicit for some reason in the next declaration
 universe u_5
+variable {k : Type _} [CommRing k] (R : Type _) [Ring R] [Algebra k R] {M : Type _}
+  [AddCommMonoid M] [Module k M] {ι : Type u_5} (b : Basis ι k M)
```

Comment:
Can we just make all the universes explicit to play it safe?
```suggestion
-- porting note: need to make a universe explicit for some reason in the next declaration
universe uk uR uM uι
variable {k : Type uk} [CommRing k] (R : Type uR) [Ring R] [Algebra k R] {M : Type uM}
  [AddCommMonoid M] [Module k M] {ι : Type uι} (b : Basis ι k M)
```
This also results in error messages that are easier to understand when there are universe problems.

Reply:
I'm not sure how much difference it makes but sure why not.